nautobot_netbox_importer/tests/fixtures/3.0/summary.json

```diff
@@ -2399,7 +2399,7 @@
                     "importer": "value_importer",
                     "definition": "color",
                     "from_data": true,
-                    "is_custom": true,
+                    "is_custom": false,
```

Comment:
Unrelated change?

Reply:
This file is generated, don't want to change it manually.

Noticed this `color` field flapping. Want to fix it in a separate PR, not a real issue now.